src/applications/vaos/tests/mocks/setup.js

```diff
@@ -167,3 +182,64 @@ export async function setPreferredDate(store, preferredDate) {
 
   return router.push.firstCall.args[0];
 }
+
+export function setupExpressCareMocks({
+  facilityId = '983',
+  isWindowOpen = true,
```

Comment:
I don't think we have enough of these to call it a convention, but I know for the eligibility mock function, I defaulted to values that disabled functionality, rather than defaulting to things being enabled. Might be useful to be consistent there. 

Mainly I want the tests themselves to show info relevant to the test and hide irrelevant setup code, so we don't have to flip back to these functions to see what's on or off.